src/wasm/thorvgwasm.cpp

```diff
@@ -0,0 +1,172 @@
+#include <thorvg.h>
+
```

Comment:
For less code, 

using namespace std; 
using namespace tvg; 

and remove all other std & tvg prefixes here.